<?xml version="1.0" encoding="UTF-8"?>
<beans xmlns="http://www.springframework.org/schema/beans"
       xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
       xmlns:context="http://www.springframework.org/schema/context"
       xmlns:mvc="http://www.springframework.org/schema/mvc"
       xmlns:aop="http://www.springframework.org/schema/aop"
       xsi:schemaLocation="http://www.springframework.org/schema/beans
           http://www.springframework.org/schema/beans/spring-beans-3.0.xsd
           http://www.springframework.org/schema/context
           http://www.springframework.org/schema/context/spring-context-3.0.xsd
           http://www.springframework.org/schema/mvc
           http://www.springframework.org/schema/mvc/spring-mvc.xsd http://www.springframework.org/schema/aop http://www.springframework.org/schema/aop/spring-aop.xsd
http://www.springframework.org/schema/context ">

    <import resource="mvc.xml"/>
    <!--<import resource="database.xml"/>-->


    <!-- Настраиваем область поиска для Spring MVC -->
    <context:component-scan base-package="social_help"/>

    <mvc:resources mapping="/**" location="/"/>
    <mvc:annotation-driven/>


    <!--Убрал на время-->
    <!--<aop:aspectj-autoproxy/>-->

    <!--Junit testing  NOW-->

    <!--<bean id="exceptionResolver" class="org.springframework.web.servlet.handler.SimpleMappingExceptionResolver">-->
    <!--<property name="exceptionMappings">-->
    <!--<props>-->
    <!--<prop key="net.petrikainulainen.spring.testmvc.todo.exception.TodoNotFoundException">error/404</prop>-->
    <!--<prop key="java.lang.Exception">error/error</prop>-->
    <!--<prop key="java.lang.RuntimeException">error/error</prop>-->
    <!--</props>-->
    <!--</property>-->
    <!--<property name="statusCodes">-->
    <!--<props>-->
    <!--<prop key="error/404">404</prop>-->
    <!--<prop key="error/error">500</prop>-->
    <!--</props>-->
    <!--</property>-->
    <!--</bean>-->


</beans>